### PR TITLE
Install all .NET in one step

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -53,22 +53,13 @@ jobs:
           MSSQL_SA_PASSWORD: Password12!
 
     steps:
-    - uses: actions/checkout@v2
```
Comment (lahma) [inline .github/workflows/build.yml:56]: You need to keep the checkout

**Comment by lahma** [comment]
Did I trigger a merge with a comment 🤯

**Comment by sebastienros** [comment]
What happened here?

**Comment by sebastienros** [comment]
> enabled auto-merge

**Comment by sebastienros** [comment]
And now:

> MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.

**Comment by sebastienros** [comment]
Resolved

**Pull request merged**